redash/controllers.py

```diff
@@ -80,8 +84,8 @@ def login():
                            analytics=settings.ANALYTICS,
                            next=request.args.get('next'),
                            username=request.form.get('username', ''),
-                           show_google_openid=settings.GOOGLE_OAUTH_ENABLED)
-
+                           show_google_openid=settings.GOOGLE_OAUTH_ENABLED,
+                           show_saml_login=settings.SAML_LOGIN_ENABLED)
```

Comment:
Good point. I didn't include the section in login.html to redirect to the saml login page. I've just updated the PR.